docs/content/integrations/airbyte.mdx

```diff
@@ -0,0 +1,136 @@
+```
+
+The `load_assets_from_airbyte_project` function parses the YAML metadata, generating a set of software-defined assets which reflect each of the data streams synced by your connections. Each connection has an associated [operation](https://docs.dagster.io/concepts/ops-jobs-graphs/ops#ops) which triggers a sync of that connection.
+
+#### Adding a resource
+
+Assets loaded from Airbyte require an `airbyte_resource`, which defines how to connect and interact with your Airbyte instance.
+
+You can configure this resource and add it to your Airbyte assets by doing the following:
+
+```python startafter=start_airbyte_project_config endbefore=end_airbyte_project_config file=/integrations/airbyte/airbyte.py dedent=4
+from dagster_airbyte import airbyte_resource, load_assets_from_airbyte_project
+
+from dagster import with_resources
+
+airbyte_assets = with_resources(
+    load_assets_from_airbyte_project(project_dir="path/to/airbyte/project"),
+    {
+        "airbyte": airbyte_resource.configured(
+            {
+                "host": "localhost",
+                "port": "8000",
+            }
+        )
+    },
+)
+```
+
+### Loading Airbyte assets from Airbyte instance
+
+To load Airbyte assets into Dagster from a live Airbyte instance, you will need to configure an Airbyte resource which defines how to connect to that instance. Here, the Airbyte instance is treated as the source of truth.
+
+```python startafter=start_load_assets_from_airbyte_instance endbefore=end_load_assets_from_airbyte_instance file=/integrations/airbyte/airbyte.py dedent=4
+from dagster_airbyte import airbyte_resource, load_assets_from_airbyte_instance
+
+airbyte_instance = airbyte_resource.configured(
+    {
+        "host": "localhost",
+        "port": "8000",
+    }
+)
+airbyte_assets = load_assets_from_airbyte_instance(airbyte_instance)
+```
+
+The `load_assets_from_airbyte_instance` function retrieves a list of connection definitions via the Airbyte API, generating a set of software-defined assets which reflect each of the data streams synced by your connections. Each connection has an associated [operation](https://docs.dagster.io/concepts/ops-jobs-graphs/ops#ops) which triggers a sync of that connection.
+
+The assets produced by this function are automatically bound the passed `airbyte_resource` to use at runtime.
+
+---
+
+## Scheduling Airbyte syncs
+
+Once you have Airbyte assets, you can define a job that runs some or all of these assets on a schedule, triggering the underlying Airbyte sync:
+
+```python startafter=start_schedule_assets endbefore=end_schedule_assets file=/integrations/airbyte/airbyte.py dedent=4
+from dagster import ScheduleDefinition, define_asset_job, repository
+
+run_everything_job = define_asset_job("run_everything", selection="*")
+
+# only my_model and its children
+run_something_job = define_asset_job("run_something", selection="my_connection*")
```

Comment:
Does this work? There won't necessarily be an asset with the name of the connection, right?

Reply:
Right, it's probably more accurate to use something like `my_stream*`